package.json

```diff
@@ -7,7 +7,7 @@
     "node": ">=10"
   },
   "scripts": {
-    "test": "tap",
+    "test": "tap --test-ignore=test/constants.js",
```

Comment:
this shouldn't be necessary. in any case let's whitelist instead of blacklisting, if it's necessary

Reply:
I'm not sure it's 100% necessary, I just didn't like the output in the console telling me that it skipped over these files due to no tests found, so I added the whitelist instead `--files=test/**/*.spec.js`